openapi/index.yaml

```diff
@@ -165,6 +184,8 @@ definitions:
       - hpan
       - activation_periods
       - enabled
+      - channel
+      - channel_descr
```

Comment:
Are these fields always present? We can put it optional to avoid decoding errors

Reply:
ok done, maybe